vars/kola.groovy

```diff
@@ -7,8 +7,6 @@
 //    arch:               string   -- the target architecture
 //    cosaDir:            string   -- cosa working directory
 //    parallel:           integer  -- number of tests to run in parallel (default: # CPUs)
-//    skipBasicScenarios  boolean  -- skip basic qemu scenarios
-//    skipSecureBoot      boolean  -- skip secureboot tests
```

Comment:
We should probably keep `skipSecureBoot`, except that now it would do something like `--denylist-test *.uefi-secure`. This is used by https://github.com/coreos/fedora-coreos-pipeline/blob/0525628c6e8d7fc5ff46bc33774e88b59ced7f49/docs/config.yaml#L30-L31 and is useful when building with a kernel that's not signed (of course, you'll be able to add it to the `kola-denylist.yaml` with the new work, but it's nice to be able to control it from the pipecfg too to not require forking the repo just for that).

Reply:
Ah ok thank you :)